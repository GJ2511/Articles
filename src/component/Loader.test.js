import React from 'react';

import Loader from './Loader';

describe('<Loader tests />', () => {
    test('match snapshot', () => {
        expect(shallow(<Loader />)).toMatchSnapshot();
    });
});
