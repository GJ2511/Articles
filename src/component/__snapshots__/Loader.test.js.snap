// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader tests /> match snapshot 1`] = `
<div
  className="d-flex justify-content-center"
>
  <div
    className="spinner-border"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
